README.md

```diff
@@ -40,7 +40,7 @@ structure Expect:
     val error: exn -> (unit -> 'a) -> expectation
     datatype expectation = Fail of string * string | Pass
     val falsy: bool -> expectation
-    val nearTo: real -> real -> expectation
+    val nearTo: real -> real -> real -> expectation
```

Comment:
I would suggest to update the README differently: I would actually delete the whole chapter "### testlib.sml" since it does not contain usefull information which cannot be obtained elsewhere. Moreover one never knows if the info is outdated. The signature is easy to get from `lib/testlib.sml`, and if an explicit signature `EXPECT` was usefull I would just add it to the file (with a corresponding `structure Expect :> EXPECT =`. For usage I would just refer to the tests. On the other hand I would add the info that the `testlib.sml` files are intended to be copies of `lib/testlib.sml`, directly at the place where "### testlib.sml" was (it is written somewhere else however).

Reply:
You are probably right, but I think that is outside the scope of this PR.

Reply:
OK I am fine if you think it should not be part of this PR. My reasoning was: If I have to touch in anyway and I think it is not needed anymore and it is nothing really important then I can just delete it now.

If you like you can open an issue on that or just do another PR.